docs/versioned_docs/version-2.11.0/deprecated/releases-0x.md

```diff
@@ -0,0 +1,296 @@
+---
+id: releases-0x
+title: OpenEBS 0.x Deprecated Releases
+---
+
+## 0.9.0 - May 24 2019
+
+:::note Deprecation notice
+This release has been deprecated. Please upgrade to the latest release. See [upgrade instructions](/docs/next/upgrade.html).
```

Comment:
There is no `/next` route. Please use the new route i.e., `/docs/user-guides/upgrade`

Reply:
Please correct this in other places as well